src/heartbeat/auth.py

```diff
@@ -16,7 +16,8 @@ def _decorator(request, *args, **kwargs):
             ip = get_client_ip(request)
             if ip in auth['authorized_ips']:
                 return func(request, *args, **kwargs)
-
+        if auth.get('disable', False):
```

Comment:
This logic would make more sense above the `if 'authorized_ips' in auth`.
I don't see any reason why we should do anymore checks if the basic auth is disabled.
Also to exclude any other value(than a boolean one) from being set to the 'disable' key, a more accurate check would probably be `if auth.get('disable', False) is True:`
Thanks!